### DISCUSSION TITLE
✨ slskd

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2222] [Requests]
### Description

A modern client-server application for the [Soulseek](https://www.slsknet.org/news/) file-sharing network.

### Official Website

https://www.slsknet.org/news/

### Documentation link

https://github.com/slskd/slskd/blob/master/docs/docker.md

### Last application release & date

0.19.5 (23/12/2023)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/slskd/slskd/

### Docker image link

https://github.com/orgs/slskd/packages/container/package/slskd

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @tandy-1000

**Comment by PieterDherde** [comment]
With the new external link function it's easy to run the SLSKD Docker image outside of tipi and have easy access, but it would be nice to have it included in the traefik proxy manager.
